src/libs/EmojiUtils.js

```diff
@@ -201,7 +202,7 @@ function replaceEmojis(text, isSmallScreenWidth = false) {
     for (let i = 0; i < emojiData.length; i++) {
         const checkEmoji = emojisTrie.search(emojiData[i].slice(1, -1));
         if (checkEmoji && checkEmoji.metaData.code) {
-            let emojiReplacement = checkEmoji.metaData.code;
+            let emojiReplacement = this.getEmojiCodeWithSkinColor(checkEmoji.metaData, preferredSkinTone);
```

Comment:
How come I was not able to reproduce this on localhost? Is this caused by the diff between debug javascript files and the built ones?

Reply:
Good find, and that's quite interesting about `this.` working on Dev but not production 🤔  Maybe we need some better eslint rules to catch this in the future?

Reply:
@Beamanator That would be good. Can you please create a tracking issue for this one? The lint rule is good approach to prevent this bug but what about other unknown diff between dev and built js files?